themes/OneNord-color-theme.json

```diff
@@ -44,8 +44,8 @@
       "editorCursor.foreground": "#d8dee9",
       "editorHint.border": "#D08F7000",
       "editorHint.foreground": "#D08F70",
-      "editorIndentGuide.background": "#434c5eb3",
-      "editorIndentGuide.activeBackground": "#4c566a",
+      "editorIndentGuide.background1": "#434c5eb3",
+      "editorIndentGuide.activeBackground1": "#4c566a",
```

Comment:
`editor.IndentGuide.background` has been deprecated in favour of `editor.IndentGuide.background1`, `editor.IndentGuide.background2`...and so on